packages/office-ui-fabric-react/src/components/Panel/Panel.tsx

```diff
@@ -265,7 +262,6 @@ export class Panel extends BaseComponent<IPanelProps, IPanelState> {
     let _content = this._content;
     let height = _content.clientHeight;
     let innerHeight = _content.scrollHeight;
-
     this.setState({
       isFooterSticky: height < innerHeight ? true : false
```

Comment:
But doesn't height < innerHeight already return true/false? I meant that you could just do ` isFooterSticky: height < innerHeight ` and it would work the same. I think